witnet.toml

```diff
@@ -52,10 +52,12 @@ server_address = "127.0.0.1:21338"
 
 [consensus_constants]
 checkpoints_period = 45
-checkpoint_zero_timestamp = 1585742400
-bootstrap_hash = "00000000000000000000000000000000000000007769746e65742d302e372e33"
-genesis_hash = "15493414fde678612be9811777bb26f019af75de23daffd41974fccd784c89b2"
-mining_replication_factor = 1
+# Monday, 27-Apr-2020, 14:00 UTC
+checkpoint_zero_timestamp = 1587996000
+bootstrap_hash = "00000000000000000000000000000000000000007769746e65742d302e382e30"
+genesis_hash = "a6901661b32aed279d5c511b5edf0febb0f068927f533619271802fa49e842eb"
```

Comment:
Hey let's not PR and merge this `genesis_hash`, which is intended only for local, testing purposes. I'd rather do another PR for the final `genesis_hash` when we know it.